install/configuring_inventory_file.adoc

```diff
@@ -235,7 +235,7 @@ to `node-role.kubernetes.io/compute=true` if undefined.
 |`openshift_docker_additional_registries`
 a|{product-title} adds the specified additional registry or registries to the
 *docker* configuration. These are the registries to search.
-If the registry required to access the registry is other than `80` include the port number required.
+If the registry required to access the registry is other than `80` include the port number required in the form of `<address>:<port>`.
```

Comment:
s/include/include,
I'd s/port number required/that port number (I think it's a little clearer this way. I think that you can tell from the context that it's required.)